modules/internal/ChapelRange.chpl

```diff
@@ -16,21 +16,98 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/*
+  The ChapelRange module provides the implementation for the built in range
```

Comment:
I suggest removing the first sentence "The ChapelRange module provides the implementation ..." because it is our internal implementation detail. I envision when users click on "Range" in the contents, they get to a page that talks about ranges regardless of whether there is a module for it or not. Ditto tuples, arrays, etc.